src/lib/swapper/LifiSwapper/utils/constants.ts

```diff
@@ -1,7 +1,3 @@
-import type { SwapSource } from '@shapeshiftoss/swapper'
-import { SwapperName } from '@shapeshiftoss/swapper'
-
-export const DEFAULT_SOURCE: SwapSource[] = [{ name: SwapperName.LIFI, proportion: '1' }]
 export const MIN_AMOUNT_THRESHOLD_USD_HUMAN = 20 // arbitrary amount deemed by lifi devs to meet minimum amount across all brdiges
```

Comment:
*OCD intensifies noticing this right after merging it*
```suggestion
export const MIN_AMOUNT_THRESHOLD_USD_HUMAN = 20 // arbitrary amount deemed by lifi devs to meet minimum amount across all bridges
```